Client/Frontend/Menu/MenuConfiguration.swift

```diff
@@ -152,58 +160,58 @@ struct FirefoxMenuConfiguration: MenuConfiguration {
 extension FirefoxMenuConfiguration {
 
     private static var NewTabMenuItem: MenuItem {
-        return FirefoxMenuItem(title: NewTabTitleString, action: .OpenNewNormalTab, icon: "menu-NewTab", privateModeIcon: "menu-NewTab-pbm")
+        return FirefoxMenuItem(title: NewTabTitleString, action: MenuAction(action: AppMenuAction.OpenNewNormalTab.rawValue), icon: "menu-NewTab", privateModeIcon: "menu-NewTab-pbm")
```

Comment:
Nit: This feels very verbose. 
Is there a neater way, e.g. `typealias AppMenuAction = String`, and use static variables?